src/metadatas.rs

```diff
@@ -164,6 +168,61 @@ impl Metadata {
     }
 }
 
+pub fn compute_stats(gtfs: &gtfs_structures::RawGtfs) -> Stats {
+    Stats {
+        stops_count: gtfs.stops.as_ref().map_or(0, |stops| stops.len()),
+        stop_areas_count: counts_objects(&gtfs.stops, |s| {
+            s.location_type == gtfs_structures::LocationType::StopArea
+        }),
+        stop_points_count: counts_objects(&gtfs.stops, |s| {
+            s.location_type == gtfs_structures::LocationType::StopPoint
+        }),
+        stops_with_wheelchair_info_count: None,
```

Comment:
```suggestion
        stops_with_wheelchair_info_count: stops_with_wheelchair_info_count(gtfs),
```

Et adapter les tests j'imagine

Reply:
ah non ca ca necessite un object `GTFS` (et non `RawGTS`), vu qu'on utilise les liens entres stops pour checker ca.

Je cale un test pour que ce soit plus explicite.